nucliadb_models/src/nucliadb_models/augment.py

```diff
@@ -87,15 +228,25 @@ class AugmentedParagraph(BaseModel):
 
 
 class AugmentedField(BaseModel):
+    text: str | None = None
+
+    classification_labels: dict[str, list[str]] | None = None
```

Comment:
Is the dict to group by labelset?
I wonder if it's easier for the user to simply get a list of strings here? Same for the entities.
Maybe you could check with the API chieff.